scripts/shapes.py

```diff
@@ -1,23 +1,49 @@
+from typing import List
```

Comment:
These imports are not used. Further they are very close to existing reserved names of list.
Why do these need included?

(same commend for import in other scripts)

Reply:
See above.